src/attr/_make.py

```diff
@@ -248,8 +251,13 @@ def attributes(maybe_cls=None, these=None, repr_ns=None,
 
         ..  _slots: https://docs.python.org/3.5/reference/datamodel.html#slots
 
+    :param bool validate_types: Check that each value passed to the constructor
```

Comment:
This reduces redundancy in having to specify both a type and the `instance_of` validator for that type.